config/app.yml

```diff
@@ -0,0 +1,42 @@
+#
+# Test config
```

Comment:
Can this not live in the test directory rather than the root to stop confusion?

Reply:
I tried that first - but the logger is looking for:

var config = require('konfig')({ path: 'config' });

If you know a way of getting the logger to pick up the config from elsewhere - then please advise and I'll change.

Reply:
Is this just for your tests?

If so you could mock the config using something like mock-require

Reply:
Yes, it's so I can start an app in the tests. I'll try mock-require.